src/xcode/ENA/TestPlans/UITests.xctestplan

```diff
@@ -89,7 +89,51 @@
       "parallelizable" : true,
       "skippedTests" : [
         "ENAUITests",
-        "ENAUITests_11_QuickActions\/testLaunchViaShortcutFromFreshInstall()"
+        "ENAUITests_00_Onboarding\/test_0002_Screenshots_OnboardingFlow_EnablePermissions_normal_S()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_invalid_test_result()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_negative_test_result()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_pending_test_result()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_positive_test_result()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_riskCardHigh_riskOneDay()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_riskCardLow_14days()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_riskCardLow_riskNoDays()",
+        "ENAUITests_01a_Home\/test_screenshot_homescreen_riskCardLow_riskOneDay()",
+        "ENAUITests_01b_Statistics\/test_screenshot_local_statistics_card()",
+        "ENAUITests_01b_Statistics\/test_screenshot_statistics_card_titles()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_SubmitQR()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_SubmitTAN()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_SymptomsOptionYes()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_TestCertificateScreen()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_symptoms_onset_date_option()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_certificate_test_result_negative()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_available()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_pending()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_positive()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_AddTwoPersonsTwoLocations()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_ContactJournalInformation()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_EditLocationScreen()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_EditPersonScreen()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_TwoPersonsOneLocationAndMessages()",
+        "ENAUITests_09_TraceLocations\/test_screenshots_of_traceLocation_print_flow()",
+        "ENAUITests_10_CheckIns\/test_screenshot_WHEN_scan_QRCode_THEN_checkin_and_checkout()",
+        "ENAUITests_11_QuickActions\/testLaunchViaShortcutFromFreshInstall()",
```

Comment:
this one is a screenshot-test, too? the name gives no idea about it. Perhaps rename it?

Reply:
Nope its not a Screenshot but take a look into the test:
`This test is INTENTIONALLY disabled`